ding2.info

```diff
@@ -21,7 +21,7 @@ dependencies[] = field_ui
 ; Helper modules.
 dependencies[] = module_filter
 dependencies[] = menu_position
-dependencies[] = cookiecontrol
+dependencies[] = eu-cookie-compliance
```

Comment:
Is this the correct name for the module? Naming on drupal.org and in the module code changes from `eu-cookie-compliance` and `eu_cookie_compliance`.